SumOfNumbers/src/cst1201/NumberSummer.java

```diff
@@ -1,14 +1,44 @@
 package cst1201;
 
+import java.util.Scanner;
+
+/**
+ *
+ * Devya Gurung
+ */
+
 public class NumberSummer {
 
+    private static int number;
+
     public static void main(String[] args) {
         /*
-        Write a program that asks the user for a positive nonzero integer value. 
-        The program should use a loop to get the sum of all the integers from 1 
-        up to the number entered. For example, if the user enters 50, the loop 
-        will find the sum of 1, 2, 3, 4, ... 50.
+         Write a program that asks the user for a positive nonzero integer value. 
+         The program should use a loop to get the sum of all the integers from 1 
+         up to the number entered. For example, if the user enters 50, the loop 
+         will find the sum of 1, 2, 3, 4, ... 50.
          */
+
+        // ask user for a positive nonzero integer value
+        System.out.print("Please enter a positive non zero integer: ");
+        Scanner scan = new Scanner(System.in);
+        int maxNumber = scan.nextInt();
+        
+               
+        if (maxNumber > 0) {
+            int sum = 0;
+            int numberToAdd = 1;
+
+            while (numberToAdd <= maxNumber) {
+                sum += numberToAdd;
+                System.out.print(++numberToAdd + ", ");
```

Comment:
You don't need to print it. I was just showing you as an example. But, you **do** need the `++`.